design-proposals/klip-8-interactive-queries.md

```diff
@@ -0,0 +1,143 @@
+# KLIP 8 - Interactive Queries
+
+**Author**: derekjn | 
+**Release Target**: 5.4+ | 
+**Status**: In Discussion | 
+**Discussion**: [#530](https://github.com/confluentinc/ksql/issues/530)
```

Comment:
The discussion is more generally on the KLIP PR. The comments on the KLIP are more link the discussion on the email thread for Apache KIPs, so to mirror KIPs this would be more:

```suggestion
**Discussion**: [#530](https://github.com/confluentinc/ksql/pull/3117)
```

Though I guess you can get to the KLIP from the Issue...

Reply:
Thanks @big-andy-coates, I will link to the issue separately as I think it still provides important context.